packages/headless/src/ssr-next/commerce/engine/commerce-engine.ssr.test.ts

```diff
@@ -126,6 +137,7 @@ describe('Commerce Engine SSR', () => {
     it('should only require cart options', () => {
       expect(() =>
         recommendationEngineDefinition.fetchStaticState({
+          navigatorContext: mockNavigatorContext,
```

Comment:
we should pass a navigatorContext **Provider**.  Not the navigator context itself.
It has to be a function since it is evaluated later